pkgs/applications/networking/instant-messengers/beeper/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, mkDerivation, fetchurl, appimageTools, imagemagick, libsecret, ... }:
+let
+  pname = "beeper";
+  version = "3.62.20";
+  src = fetchurl {
+    url = "${pname}-${version}.AppImage::https://download.beeper.com/linux/appImage/x64";
+    sha256 = "5389e24ff3cef9acc6f137d24d37e9ef319c865ef81fa3337d407f927f087e31";
```

Comment:
Is it possible to change `sha256 =` to the more recommended `hash =`

Reply:
Done :D